### PR TITLE
Parse Cli Args

### PR DESCRIPTION
**Pull request by lpmusicon**
I used [cxxopts](https://github.com/jarro2783/cxxopts) lib as it was a little nicer to work with. Not sure if lib directory was needed (where should I put external libs?)

### CODE REVIEW

File: src/cli/main.cpp [hunk 1]
```diff
@@ -36,22 +39,32 @@ int main(int argc, const char *argv[]) {
...
-    }
-    mcpx_path = argv[1];
-    bios_path = argv[2];
-    xbe_path = argv[3];
-	model = argv[4];
+	// Parse arguments
+	const char *mcpx_path = args["mcpx"].as<std::string>().c_str();;
+	const char *bios_path = args["bios"].as<std::string>().c_str();;
+	const char *xbe_path = args["xbe"].as<std::string>().c_str();;
+	const char *model = args["model"].as<std::string>().c_str();;
```
Comment (StrikerX3) [inline src/cli/main.cpp]: Nitpick: double semicolons on these four lines.

**Comment by StrikerX3** [comment]
The lib folder you created is fine for the purposes of including cxxopts (which was another alternative I was studying at the time) and possibly other single-header libraries like that.

Don't forget to create a CMakeLists.txt file under that folder and include it in the parent folder's CMakeLists too.

Other than that and the nitpick, looks good to me.

**Comment by lpmusicon** [comment]
Added CMakeLists, fixed double semicolon and updated readme. Should be all good now

**Comment by StrikerX3** [comment]
Great! Merging now.

**Pull request merged**